src/main/java/frc/team4909/robot/LIDAR.java

```diff
@@ -1,79 +1,84 @@
-package frc.team4909.robot;
```

Comment:
is there a need to comment all of this out? if you're not using it, it should be fine, ideally

Reply:
Code will be kept for future years.